packages/settings/questionsSelection.stable.ts

```diff
@@ -98,6 +99,12 @@ export const collection: QuestionCollection = [
     contents: [huffmanCoding, ReverseHuffmanCoding],
     topics: ["algorithms"],
   },
+  {
+    slug: "setbuilder",
+    name: { en: "Set Builder Notation", de: "Mengenschreibweise" },
```

Comment:
The question group should just be named "Sets" or something like this. the same as the question itself.